drivers/dac/ad9144/ad9144.c

```diff
@@ -363,6 +364,72 @@ static int32_t ad9144_pll_setup(struct ad9144_dev *dev,
 	return ret;
 }
 
+int32_t ad9144_set_nco(struct ad9144_dev *dev, int32_t f_carrier_khz,
```

Comment:
f_carrier_khz should it be uint32 ?

Reply:
Using the SEL_SIDEBAND register, you can use negative or positive modulation.